config/theme-tabler.php

```diff
@@ -36,6 +36,12 @@
     // fluid
     // fluid_vertical
 
+    /**
+     * Pick a login page layout.
+     * Possible values: default, illustration, cover
+     */
+    'layout_login' => 'default', // default, illustration, cover
+
     /**
      * 2nd Layer of customization
      *
```

Comment:
It's very important for us (Backpack) to have the footer on all Login layouts. Why? Because most devs do NOT remove the "powered by backpack" logo. And we have a link to our website there. This creates a simple but very effective mechanism for backlinks for us. 80% of people who use Backpack (free or paid) basically include a backlink to us. If that page is crawled by search engines (sometimes it is, sometimes it's not) then... it's free SEO for us.

So the footer should ALWAYS be there, on all auth pages. It's a business decision, not a design decision.

But from a design perspective... yeah it's useless. So it should be as un-intrusive as possible. I'd go with transparent background, like we have it now - https://demo.backpackforlaravel.com/admin/login